install/swiftly-install.sh

```diff
@@ -330,18 +330,27 @@ if ! has_command "curl" ; then
     exit 1
 fi
 
-if ! verify_getopt_install ; then
-    echo "Error: getopt must be installed from the util-linux package to run swiftly-install"
-    exit 1
+IS_MACOS="false"
+if has_command "sw_vers"; then
```

Comment:
Interesting way to recognise a Mac, is this standard?

Reply:
I've switched this over to uname.